### ISSUE TITLE
Onde está o Modsulo 5, o Curso foi em vão?

### ISSUE THREAD

**Issue by robeliomoura** [gustavoguanabara/html-css#556]
O módulo 5 só tem duas aulas, agora o aluno fica sem rumo sem saber o que faltou estudar, tem que começar outro curso do zero?

**Comment by BenjamimCS** [comment]
Acho que você ainda não entendeu a proposta do Curso Em Vídeo. Além disso, você não sabe o que estudar depois de estudar 4 módulos? Não foque em curso, ele é apenas outro meio de aprender algo, há outros: pesquise mais na área em que estuda. Não faz o menor sentido começar um outro do zero com esse conhecimento que você já tem, comece a ver o que será ensinado nesse módulo e põe pra frente.

Se você ainda não sabe o que estudar ou quer conhecer mais de Desenvolvimento Web Front-End tem o site [roadmap.sh](roadmap.sh "The Developer Roadmap), te dá um bom rumo, gosto bastante desse site. Te recomendo mais ainda ver esse [vídeo de Front-End](https://youtu.be/VKmPGmFY7H4?si=O4St3IfW2obVN6ZT "A História do Front-End para Iniciantes em Programação | Séria Começando aos 40"), te faz entender mais do assunto.

Abs., bons estudos.

**Comment by Sidnei-Junior-dev** [comment]
pelo q vi, o modulo 5 esta no EstudoNauta https://www.estudonauta.com e tem que pagar pra ter acesso. ;-;

**Comment by BenjamimCS** [comment]
> pelo q vi, o modulo 5 esta no EstudoNauta https://www.estudonauta.com/cursos/ e tem que pagar pra ter acesso. ;-;

Sim, já foi liberado para assinantes. Logo, logo libera pra nóix

**Comment by filipeluisgg** [comment]
> > pelo q vi, o modulo 5 esta no EstudoNauta https://www.estudonauta.com/cursos/ e tem que pagar pra ter acesso. ;-;
> 
> Sim, já foi liberado para assinantes. Logo, logo libera pra nóix


Bom, 'logo logo' seria em um futuro próximo, ou ainda demoraria um bom tempo?

**Comment by BenjamimCS** [comment]
> > > pelo q vi, o modulo 5 esta no EstudoNauta https://www.estudonauta.com/cursos/ e tem que pagar pra ter acesso. ;-;
> > 
> > 
> > Sim, já foi liberado para assinantes. Logo, logo libera pra nóix
> 
> Bom, 'logo logo' seria em um futuro próximo, ou ainda demoraria um bom tempo?

Quis dizer em pouco tempo, mas posso ter sido impreciso. Não sei quanto tempo ele demora pra lançar no YT após lançar na plataforma.

**Comment by flameastro** [comment]
@robeliomoura Por favor, feche a issue!
